M2/Macaulay2/packages/TorAlgebra.m2

```diff
@@ -360,7 +364,7 @@ toralgdata = R -> (
 		num = n + l*t + t^2 - t^3;
 		)
 	    else (
-		if isHomogeneous R then (
+		if isHomogeneous ideal R then (
```

Comment:
Was this change necessary?  The two conditions are equivalent.  (Just asking, no need to change it back.)